apps/rebar/src/rebar_prv_manifest.erl

```diff
@@ -0,0 +1,138 @@
+%% ===================================================================
+%% Manifest Provider
+%% ===================================================================
+-module(rebar_prv_manifest).
+-behaviour(provider).
+-export([init/1,
+         do/1,
+         format_error/1]).
+
+-include_lib("providers/include/providers.hrl").
+
+-define(PROVIDER, manifest).
+-define(DEFAULT_FORMAT, erlang).
+
+-type extension() :: string().
+-type app_context() :: #{name         := binary(),
+                         src_dirs     := [file:filename()],
+                         include_dirs := [file:filename()],
+                         src_ext      := extension(),
+                         out_mappings := [#{extension := extension(), path := file:filename()}],
+                         dependencies_opts => any()}.
+-type manifest() :: #{
+                      apps := [app_context()],
+                      deps := [app_context()],
+                      otp_lib_dir := string(),
+                      source_root := string()
+                     }.
+
+-type format() :: erlang | eetf.
+
+%% ===================================================================
+%% Provider Callbacks
+%% ===================================================================
+-spec init(rebar_state:t()) -> {ok, rebar_state:t()}.
+init(State) ->
+
+  %% By default, the provider outputs the manifest to stdout, so disable logs
+  %% not to interfere.
+  ok = rebar_log:init(api, 0),
```

Comment:
The init/1 function is invoked at Rebar3 start time for _all_ providers to know which commands are available and what their dependencies are. This means that this disables logs for rebar3 universally.

This isn't a good way to go, and if you need logs from stdout, I don't think we can prevent other providers from outputting anything, particularly if you're asking for `install_deps` to run already.

The only path forward I see if you want stdout content is to pass in an argument that is going to be an optional delimiter, like `--delimiter="~~~content~~~"` which would then give you something like:

```
~~~content~~~
this is the parseable output
~~~content~~~
```
Which frankly isn't elegant, but probably workable and without interfering with other unrelated tasks.